apps/patient/public/index.html

```diff
@@ -29,7 +29,7 @@
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
+    <div id="root" style="min-height: 100vh; display: flex; flex-direction: column"></div>
```

Comment:
why is this being done here?

Reply:
makes the app always at least extend to the whole page. Lets me put a footer at the bottom of the screen

Reply:
Can we re-use FixedFooter?

Reply:
No, thats a slide in from the bottom that only sometimes shows. Is this breaking anything? 

(In general, setting your root to `min-height: full` and `flex-col` makes styling the whole app usually easier in my experience